dist/c89-compatible/Makefile

```diff
@@ -87,7 +87,7 @@ OBJS 		+= $(patsubst %.S,%.o,$(wildcard *-$(MARCH)$(VARIANT).S))
 
 include Makefile.basic
 
-CFLAGS		+= -Wno-parentheses -Wno-deprecated-declarations -Wno-\#warnings -Wno-error=cpp -Wno-cpp -g -std=gnu11 -O3
+CFLAGS		+= -Wno-parentheses -Wno-deprecated-declarations -Wno-type-limits -Wno-error=cpp -Wno-cpp -g -std=gnu11 -O3
```

Comment:
Do we really want this? Not sure if there should be a `-std=gnu11` in the c89 dir. But I don't care since we don't use this.

Reply:
I don't know much about this. I had to disable type-limits to keep GCC 11 happy.